src/pet.js

```diff
@@ -0,0 +1,78 @@
+minAge = 0;
+minHunger = 0;
+maxFitness = 10;
+
+function Pet(name) {
+    this.name = name;
+    this.age = minAge;
+    this.fitness = maxFitness;
+    this.hunger = minHunger;
+};
+
+Pet.prototype = {
+    get isAlive() {
+        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+};
+
+Pet.prototype.growUp = function() {
+        ageIncrements = 1;
+        fitnessIncrements = 3;
+        hungerIncrements = 5;
+
+        this.age += ageIncrements;
+        this.fitness -= fitnessIncrements;
+        this.hunger += hungerIncrements;
+        
+        if(!this.isAlive) {
+        throw new Error('Your pet is no longer alive')
+    }
+    };
+
+Pet.prototype.walk = function() {    
+    if(!this.isAlive) {
+            throw new Error('Your pet is no longer alive')
+        }
+        const fitnessIncreaseWalk = 4;
+        this.fitness += fitnessIncreaseWalk;
+        if(this.fitness >= maxFitness) {
+            this.fitness = 10;
+        }
+
+    
+    };
```

Comment:
This could be refactored a bit. In essence you are adding a value to the existing fitness, without allowing it go over a set limit. You can do this with Math.min, which will return the lowest of the values passed to it:
```
this.fitness = Math.min(this.fitness + 4, MAXIMUM_FITNESS)
```




Indentation here again. Also, this could be refactored a bit. In essence you are adding a value to the existing fitness, without allowing it go over a set limit. You can do this with Math.min, which will return the lowest of the values passed to it:
```
this.fitness = Math.min(this.fitness + fitnessIncreaseWalk, maxFitness)
```

```
Pet.prototype.walk = function() {    
    if (!this.isAlive) {
        throw new Error('Your pet is no longer alive')
    }
    const fitnessIncreaseWalk = 4;
    this.fitness = Math.min(this.fitness + fitnessIncreaseWalk, maxFitness)
};
```
As before, you could move the variable creation to be with the other constants since it has value in other parts of the repo too